datasets/aster-l1t.yaml

```diff
@@ -0,0 +1,81 @@
+Name: ASTER L1T Cloud-Optimized GeoTIFFs
+Description: |
+  The Advanced Spaceborne Thermal Emission and Reflection Radiometer (ASTER) Level 1
+  Precision Terrain Corrected Registered At-Sensor Radiance (AST_L1T) data contains
+  calibrated at-sensor radiance, which corresponds with the ASTER Level 1B (AST_L1B),
+  that has been geometrically corrected, and rotated to a north-up UTM projection.
+  The AST_L1T is created from a single resampling of the corresponding ASTER L1A (AST_L1A) product.
+
+  The precision terrain correction process incorporates GLS2000 digital elevation data with
+  derived ground control points (GCPs) to achieve topographic accuracy for all daytime scenes
+  where correlation statistics reach a minimum threshold. Alternate levels of correction are
+  possible (systematic terrain, systematic, or precision) for scenes acquired at night or that
+  otherwise represent a reduced quality ground image (e.g., cloud cover).
+
+  Each AST_L1T granule is converted into three different COG files based on the sensor and spatial resolution,
+  VNIR at 15m, SWIR at 30m and TIR at 90m. The metadata required to transform the digital numbers (DN) to 
+  radiance and reflectance values are also incorporated as metadata in the TIFF files.
+  The filenaming convention and the organization of bands are described
+  [here](https://github.com/awslabs/open-data-docs/tree/main/docs/aster-l1t).
+
+Documentation: https://github.com/awslabs/open-data-docs/tree/main/docs/aster-l1t
```

Comment:
Hi @piyushrpt it doesn't look like the documentation is live. Can you post an active link to this?

Reply:
There is a PR open on the open-data-docs repo here:
https://github.com/awslabs/open-data-docs/pull/22

The source can be found here:
https://github.com/descarteslabs/open-data-docs/blob/feat/aster-l1t/docs/aster-l1t/README.md

Reply:
Great I just merged the docs PR so that link now works.